### ISSUE TITLE
index.js is unconventional 

### ISSUE THREAD

**Issue by andreykul** [bellangerq/the-shorter-the-better#4]
You are suppose to have a javascript file per controller rather than per action.
I would also recommend converting to coffeescript.

**Comment by bellangerq** [comment]
Yeah you're right. Thanks, I like those non-code related tips that make the difference.
